src/v/cluster/rm_partition_frontend.cc

```diff
@@ -123,8 +123,25 @@ ss::future<begin_tx_reply> rm_partition_frontend::begin_tx(
 
         begin_tx_reply result;
         if (leader == _self) {
+            vlog(
+              txlog.trace,
+              "dispatching name:begin_tx, ntp:{}, pid:{}, tx_seq:{} timeout:{} "
```

Comment:
Maybe replace dispatching to executing? I was a little bit confused

Reply:
done